configure.ml

```diff
@@ -16,8 +16,9 @@ let coq_macos_version = "8.7.90" (** "[...] should be a string comprised of
 three non-negative, period-separated integers [...]" *)
 let vo_magic = 8791
 let state_magic = 58791
-let distributed_exec = ["coqtop";"coqc";"coqchk";"coqdoc";"coqworkmgr";
-"coqdoc";"coq_makefile";"coq-tex";"gallina";"coqwc";"csdpcert";"coqdep"]
+let distributed_exec =
+  ["coqtop.opt"; "idetop.opt"; "queryworkertop.opt"; "proofworkertop.opt"; "tacworkertop.opt";
```

Comment:
again the `.opt`

Reply:
This is for the build-time signing [which is a noop anyways? ] so this needs to be kept in sync with the actual rule, and it will follow the renaming / copy later.